cli/azd/pkg/input/console.go

```diff
@@ -84,6 +107,118 @@ func (c *AskerConsole) Message(ctx context.Context, message string) {
 	}
 }
 
+func (c *AskerConsole) MessageUx(ctx context.Context, message string, format MessageUxType) {
+	formattedText, err := addFormat(message, format)
+	// Message and MessageUx don't return errors. Let's log the error and use the original message on error
+	if err != nil {
+		log.Printf("Failed adding format for MessageUx: %s. Using message with no ux format", err.Error())
+		c.Message(ctx, message)
+		return
+	}
+
+	c.Message(ctx, formattedText)
+}
+
+func addFormat(message string, format MessageUxType) (withFormat string, err error) {
+	switch format {
+	case Title:
+		withFormat = fmt.Sprintf("\n%s\n", message)
+	case ResultSuccess:
+		withFormat = output.WithSuccessFormat("\n%s: %s", "SUCCESS", message)
+	case ResultError:
+		withFormat = output.WithErrorFormat("\n%s: %s", "ERROR", message)
+	default:
+		return withFormat, fmt.Errorf("Unknown UX format type")
+	}
+
+	return withFormat, nil
+}
```

Comment:
This is a good addition but still feel like a Band-Aid when we need surgery.  Lets get a a point where we can design what an cmd/action looks like all up and forces the UX dev to build things a single way.

Reply:
We forced all `commands` in the past to use the `build` function.  The initial intention was to re-use common code and keep a common foundation across commands.
Later, we learned that each command starts becoming unique.

The commands and actions must be independent.

Also, creating a new command/action should not start by designing the UX.  The Action should be driven by its behavior and what it is meant to do.  The UX presentation should be easily added or removed.  Or even independent from other commands.